axlearn/experiments/text/gpt/fuji.py

```diff
@@ -417,6 +418,36 @@ def get_trainer_kwargs(
                     "gpu-(p5.48xlarge|p4de.24xlarge)-(512|1024)",
                     mesh_shape_from_axes(data=-1, fsdp=128),
                 ),
+                (
+                    "neuron-(trn2|trn2n).48xlarge-64",
+                    ChainConfigModifier.default_config().set(
+                        config_modifiers=[
+                            MeshShapeModifier.default_config().set(
+                                mesh_shape=mesh_shape_from_axes(fsdp=-1, model=4)
+                            ),
+                            RematSpecModifier.default_config().set(
+                                remat_policies={
+                                    "model.decoder.transformer.layer": RematSpec(
+                                        prevent_cse=True,
+                                        policy=config_for_function(
+                                            _save_and_offload_only_these_names_regex
+                                        ).set(
+                                            # pylint: disable=anomalous-backslash-in-string
+                                            names_which_can_be_saved=r"(TransformerAttentionLayer\.residual_add"  # pylint: disable=C0301
+                                            "|.*\.?(k|q|v)_proj"
+                                            "|.*\.?linear1_[01]"
+                                            "|TransformerFeedForwardLayer\.mlp_residual)",
```

Comment:
Consider defining it as a constant so that it can be reused across configs?

Reply:
Added them as a Enum class